client/src/ui/Form.jsx

```diff
@@ -212,7 +232,12 @@ export default class Form extends Component {
       : //else
       [] //set to this
     const isCovid = this.state.isCovid;
-
+    const grade = this.state.selectedGrade.length !== 0 ? //If length is not 0
+      this.state.selectedGrade.label : ''   
+    const major = this.state.selectedMajors.length !== 0 ? //If length is not 0
+      this.state.selectedMajors.map(major => { return major.label }) //set to this
+      : //else
+      [] //set to this    
```

Comment:
if the `selectedMajors.length === 0`, it seems like it would already be the empty array `[]`, so I think this ternary might be redundant